google_reauth/challenges.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+import base64
+import sys
+
+import pyu2f.convenience.authenticator
+import pyu2f.errors
+import pyu2f.model
+import six
+
+from google_reauth import _helpers
+
+
+REAUTH_ORIGIN = 'https://accounts.google.com'
+
+
+def build_challenges():
```

Comment:
On second read, this could just be a constant, no?

```
AVAILABLE_CHALLENGES = {
    challenge.name: challenge
    for challenge in [
        SecurityKeyChallenge,
        PasswordChallenge
    ]
}
```